i18n/language/language.go

```diff
@@ -39,41 +48,65 @@ func (l *Language) MatchingTags() []string {
 // Parse returns a slice of supported languages found in src or nil if none are found.
 // It can parse language tags and Accept-Language headers.
 func Parse(src string) []*Language {
```

Comment:
This function is now completely unused, so I would want to remove it, but that would be an unnecessary backward incompatible change. It seems like `ParseFirst` can/should just call `Parse` and get the first element returned.